app/configurator/components/field.tsx

```diff
@@ -818,8 +825,20 @@ export const MultiFilterFieldColorPicker = ({
   label: string;
   symbol: LegendSymbol;
 }) => {
-  const { color, checked, palette, onChange } =
-    useMultiFilterColorPicker(value);
+  const user = useUser();
+
+  const { data: customColorPalettes } = useFetchData({
+    queryKey: ["colorPalettes", user?.id],
+    queryFn: getCustomColorPalettes,
+    options: {
+      enable: !!user?.id,
+    },
+  });
```

Comment:
@noahonyejese I wonder if it wouldn't make sense to extract this to a `useUserPalettes` hook? I think it's also used in different places in the application right now, and probably would make sense even to make sure we use the same `queryKey` across them 🤔